scripts/entity/components/movement_components/player_movement_component.gd

```diff
@@ -0,0 +1,62 @@
+class_name PlayerMovementComponent extends BaseMovementComponent
+## The movement implementation for a human player.
+
+## The max horizontal speed the entity can move.
+@export var MAX_SPEED: int = 200
+## The jump velocity of the entity.
+@export var JUMP_VELOCITY: int = -300
+## The horizontal movement acceleration for the entity.
+@export var ACCELERATION: int = 1500
+## The time it takes to fully decelerate from horizontal input in seconds.
+@export_range(0, 10, 0.1, "or_greater") var DECELERATION_TIME: float = 0.2
+
+## The current input direction.
+var current_input_direction: float = 0
+## How fast the character decelerates after loss of input.
+var deceleration: float = MAX_SPEED / DECELERATION_TIME
```

Comment:
Maybe could be a const and named as such, since this doesn’t change

Reply:
It doesnt change but it cant be a constant. the values would have to be a number or constant I believe for it to be valid.